### PR TITLE
MMCA 4561 - Update Play version from 2.9 to Play 3.0

### PR DESCRIPTION
**Pull request by HariHmrc**

### CODE REVIEW

**Comment by HariHmrc** [comment]
Regarding PR Bot's comment "uk.gov.hmrc.http-verbs-play-30 can be removed - its already provided by uk.gov.hmrc.bootstrap-test-play-30",
We have not explicity declated using "http verbs" in application, so, can we ignored for now

File: build.sbt
Comment (tyagikumarvineet) [inline build.sbt:1]: Could you please update bootstrapVersion to 8.6?

File: project/AppDependencies.scala [hunk 1]
```diff
@@ -2,23 +2,21 @@ import sbt.*
...
     "uk.gov.hmrc" %% "tax-year" % "4.0.0"
   )
 
   val test: Seq[ModuleID] = Seq(
-    "uk.gov.hmrc" %% "bootstrap-test-play-29" % bootstrapVersion % Test,
+    "uk.gov.hmrc" %% "bootstrap-test-play-30" % bootstrapVersion % Test,
     "org.jsoup" % "jsoup" % "1.17.2" % Test,
-    "com.vladsch.flexmark" % "flexmark-all" % "0.64.8" % Test,
-    "org.scalatestplus.play" %% "scalatestplus-play" % "7.0.1" % Test,
     "org.mockito" %% "mockito-scala-scalatest" % "1.17.30" % Test
```
Comment (tyagikumarvineet) [inline project/AppDependencies.scala:20]: Better to update to version 1.17.31

**Comment by tyagikumarvineet** [comment]
Better to amend the PR title like below

 MMCA 4561 - Update Play from 2.9 to 3.0   

or

MMCA 4561 - Update Play version from 2.9 to 3.0

**Review by tyagikumarvineet — APPROVED** [review]

**Pull request merged**